authentication/managing_cloud_provider_credentials/cco-mode-gcp-workload-identity.adoc

```diff
@@ -0,0 +1,126 @@
+   "credential_source": {
+      "file": "<path_to_token>", <4>
+      "format": {
+         "type": "text"
+      }
+   }
+}
+----
+<1> The credential type is `external_account`.
+<2> The target audience is the GCP Workload Identity provider.
+<3> The resource URL of the service account that can be impersonated with these credentials.
+<4> The path to the service account token inside the pod. By convention, this is `/var/run/secrets/openshift/serviceaccount/token` for {product-title} components.
+
+//Supertask: Installing an OCP cluster configured for manual mode with GCP Workload Identity
+[id="gcp-workload-identity-mode-installing"]
+== Installing an {product-title} cluster configured for manual mode with GCP Workload Identity
+
+To install a cluster that is configured to use the Cloud Credential Operator (CCO) in manual mode with GCP Workload Identity:
+
+. xref:../../authentication/managing_cloud_provider_credentials/cco-mode-gcp-workload-identity.adoc#cco-ccoctl-configuring_cco-mode-gcp-workload-identity[Configure the Cloud Credential Operator utility].
+. xref:../../authentication/managing_cloud_provider_credentials/cco-mode-gcp-workload-identity.adoc#cco-ccoctl-creating-at-once_cco-mode-gcp-workload-identity[Create the required GCP resources].
+. xref:../../authentication/managing_cloud_provider_credentials/cco-mode-gcp-workload-identity.adoc#sts-mode-installing-manual-run-installer_cco-mode-gcp-workload-identity[Run the {product-title} installer].
+. xref:../../authentication/managing_cloud_provider_credentials/cco-mode-gcp-workload-identity.adoc#sts-mode-installing-verifying_cco-mode-gcp-workload-identity[Verify that the cluster is using short-lived credentials].
+
+[NOTE]
+====
+Because the cluster is operating in manual mode when using GCP Workload Identity, it is not able to create new credentials for components with the permissions that they require. When upgrading to a different minor version of {product-title}, there are often new GCP permission requirements. Before upgrading a cluster that is using GCP Workload Identity, the cluster administrator must manually ensure that the GCP permissions are sufficient for existing components and available to any new components.
+====
+
+//Task part 1: Configuring the Cloud Credential Operator utility
+include::modules/cco-ccoctl-configuring.adoc[leveloffset=+2]
+
+//Task part 2: Creating the required GCP resources all at once
+include::modules/cco-ccoctl-creating-at-once.adoc[leveloffset=+2]
+
+//Task part 3: Run the OCP installer
+include::modules/sts-mode-installing-manual-run-installer.adoc[leveloffset=+2]
+
+//Task part 4: Verify that the cluster is using short-lived credentials
+include::modules/sts-mode-installing-verifying.adoc[leveloffset=+2]
+
+////
+//Doesn't appear to be a card for this work in 4.10?
+[id="gcp-workload-identity-mode-upgrading"]
+== Upgrading an {product-title} cluster configured for manual mode with GCP Workload Identity
```

Comment:
As discussed lets include upgrade docs in 4.11 as there are no existing clusters with gcp workload identity enabled. But this should be exactly same as AWS STS.